packages/docs/docs/getting-started/quickstart.mdx

```diff
@@ -44,7 +53,13 @@ we're using throughout this documentation.
 
 :::
 
-To complete the scaffolding process, run the following command in your newly
+A new folder with the title of your project should appear in the directory 
+that you selected.
+
+Next, change into your newly created project directory.
+(Ex] cd my-first-animation)
```

Comment:
You can use a code block here:
````suggestion
Next, change into your newly created project directory. For example:
```bash
cd my-first-animation
```
````

It will make it look like a console:
```bash
cd my-first-animation
```